README.md

```diff
@@ -5,21 +5,21 @@ The intent is to provide an easy way to integrate with the AWS ecosystem to proc
 It also provides a way to implement functions in languages not supported by Pulsar Functions but supported by AWS Lambda.
 
 The function requires Pulsar 2.11+ or Luna Streaming 2.10+ to run.
+
 The function doesn't support PROTOBUF and PROTOBUF_NATIVE messages at the moment.
 
 ## Configuration
 
 The Function must be configured with the following parameters:
 
-| Name | Type|Required | Default | Description
-|------|----------|----------|---------|-------------|
-| `awsEndpoint` |String| false | " " (empty string) | AWS Lambda end-point URL. It can be found at [here](https://docs.aws.amazon.com/general/latest/gr/lambda-service.html). |
-| `awsRegion` | String| true | " " (empty string) | Supported AWS region. For example, us-west-1, us-west-2. |
-| `awsCredentialPluginName` | String|false | " " (empty string) | Fully-qualified class name of implementation of `AwsCredentialProviderPlugin`. |
-| `awsCredentialPluginParam` | String|true | " " (empty string) | JSON parameter to initialize `AwsCredentialsProviderPlugin`. |
-| `lambdaFunctionName` | String|true | " " (empty string) | The Lambda function that should be invoked by the messages. |
-| `synchronousInvocation` | Boolean|true | true | `true` means invoking a Lambda function synchronously. <br>`false` means invoking a Lambda function asynchronously. |
-
+| Name                       | Type    | Required | Default           | Description                                                                                                                                                                                                                                                                                                                     |
+|----------------------------|---------|----------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `awsEndpoint`              | String  | false    | "" (empty string) | The AWS Lambda endpoint URL. If left blank, the default AWS Lambda endpoint URL will be used.                                                                                                                                                                                                                                   |
```

Comment:
I think we can completely drop the awsEndpoint, cannot think of a use case where the user wants a different endpoint that what is decided by the provided region.

Reply:
It's necessary at least  when using Localstack in tests.
Also for [FIPS](https://aws.amazon.com/compliance/fips/#FIPS_Endpoints_by_Service) endpoints